protocol-definitions/custom/Custom.yaml

```diff
@@ -1136,3 +1136,7 @@ customTypes:
         type: SqlSummary
         nullable: true
         since: 2.5
+      - name: userCancelled
```

Comment:
@viliam-durina do we want to add this information to `JobAndSqlSummary`? It is not necessary, but can be helpful and more constistent.  If so, it should also be added to `JobSummary`

Reply:
`getJobSummaryList` returns `JobSummary` as `Data`, so there is no definition of it in client protocol